content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md

```diff
@@ -348,6 +348,13 @@ Footnotes are not supported in wikis.
 
 **Note**: The Markdown syntax used during the beta testing period is now deprecated and will be removed. You can use the syntax as described in this section.
```

Comment:
```suggestion
**Note**: The Markdown syntax used during the beta testing period is now deprecated and will be removed. You can use the current syntax as described in this section.
```

I know this is outside the scope of the things you changed, but we can also make a small tweak here for extra clarity 😁